.env.example

```
@@ -45,6 +45,9 @@ AWS_MANUAL_EXPORT_ACCESS_KEY_ID=<key_id>
 AWS_MANUAL_EXPORT_SECRET_ACCESS_KEY=<secret_key>
 AWS_MANUAL_EXPORT_BUCKET=<bucket_name>
 
+# Minutes the EA area lookup job should run for
+EA_AREA_LOOKUP_RUN_FOR=60
```

In FRAE its the water management area, in WEX its the Administrative boundary area.

So to keep things consistent across the 2 services I'd just go with

```suggestion
AREA_LOOKUP_RUN_FOR=60
```